index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="container">
+        <div id="template1">
+            <div class="bar">
+                <p>9:41</p>
+                <img class="mobile_network" src="images/header_img/tower.png" alt="network_img">
+                <img class="mobile_wifi" src="images/header_img/Wifi.png" alt="wifi_logo">
+                <img class="mobile_battery" src="images/header_img/Battery.png" alt="battery">
+            </div>
+            <header>
+                <a href="#" class="logo">My Logo</a>
+                <nav class="ham">
+                <img src="images/header_img/Union.png" alt="bar">
+                </nav>
+            </header>
+            <section>
```

Comment:
- [ ] all the content of the headline section should be aligned with flexbox, This gives us a lot of options to align all our elements, plus, it's responsive. so kindly fix it 🙏